contracts/context-config/tests/sandbox.rs

```diff
@@ -1036,7 +1036,7 @@ async fn test_deploy() -> eyre::Result<()> {
     );
 
     // Create proposal
-    let proposal_id = rng.gen();
+    let proposal_id = rand::thread_rng().gen();
```

Comment:
hm, was there a reason for this?

Reply:
just because of consistency since it was done in that way in proxy_lib tests